<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>CRAN - Package biomartr</title>
<link rel="stylesheet" type="text/css" href="../../CRAN_web.css" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<style type="text/css">
  table td { vertical-align: top; }
</style>
</head>
<body>
<h2>biomartr: Genomic Data Retrieval</h2>
<p>Perform metagenomic data retrieval and functional annotation
    retrieval. In detail, this package aims to provide users with a standardized
    way to automate genome, proteome, coding sequence ('CDS'), 'GFF', and metagenome
    retrieval from 'NCBI' and 'ENSEMBL' databases. Furthermore, an interface to the 'BioMart' database
    (Smedley et al. (2009) &lt;<a href="http://dx.doi.org/10.1186/1471-2164-10-22">doi:10.1186/1471-2164-10-22</a>&gt;) allows users to retrieve
    functional annotation for genomic loci. Users can download entire databases such
    as 'NCBI RefSeq' (Pruitt et al. (2007) &lt;<a href="http://dx.doi.org/10.1093/nar/gkl842">doi:10.1093/nar/gkl842</a>&gt;), 'NCBI nr',
    'NCBI nt' and 'NCBI Genbank' (Benson et al. (2013) &lt;<a href="http://dx.doi.org/10.1093/nar/gks1195">doi:10.1093/nar/gks1195</a>&gt;) as
    well as 'ENSEMBL' and 'ENSEMBLGENOMES' with only one command.</p>
<table summary="Package biomartr summary">
<tr>
<td>Version:</td>
<td>0.4.0</td>
</tr>
<tr>
<td>Depends:</td>
<td>R (&ge; 3.1.1)</td>
</tr>
<tr>
<td>Imports:</td>
<td><a href="http://www.bioconductor.org/packages/release/bioc/html/biomaRt.html"><span class="BioC">biomaRt</span></a>, <a href="http://www.bioconductor.org/packages/release/bioc/html/Biostrings.html"><span class="BioC">Biostrings</span></a>, <a href="../stringi/index.html">stringi</a>, <a href="../curl/index.html">curl</a>, <a href="../tibble/index.html">tibble</a>, <a href="../jsonlite/index.html">jsonlite</a>, <a href="../data.table/index.html">data.table</a> (&ge; 1.9.4), <a href="../dplyr/index.html">dplyr</a> (&ge; 0.3.0), <a href="../readr/index.html">readr</a> (&ge; 0.2.2), <a href="../downloader/index.html">downloader</a> (&ge; 0.3), <a href="../RCurl/index.html">RCurl</a> (&ge; 1.95-4.5), <a href="../XML/index.html">XML</a> (&ge; 3.98-1.1), <a href="../httr/index.html">httr</a> (&ge; 0.6.1), <a href="../stringr/index.html">stringr</a> (&ge; 0.6.2)</td>
</tr>
<tr>
<td>Suggests:</td>
<td><a href="../knitr/index.html">knitr</a> (&ge; 1.6), <a href="../rmarkdown/index.html">rmarkdown</a> (&ge; 0.3.3), <a href="../devtools/index.html">devtools</a> (&ge; 1.6.1), <a href="../testthat/index.html">testthat</a></td>
</tr>
<tr>
<td>Published:</td>
<td>2017-03-14</td>
</tr>
<tr>
<td>Author:</td>
<td>Hajk-Georg Drost</td>
</tr>
<tr>
<td>Maintainer:</td>
<td>Hajk-Georg Drost  &#x3c;&#x68;&#x67;&#x64;&#x32;&#x33;&#x20;&#x61;&#x74;&#x20;&#x63;&#x61;&#x6d;&#x2e;&#x61;&#x63;&#x2e;&#x75;&#x6b;&#x3e;</td>
</tr>
<tr>
<td>BugReports:</td>
<td><a href="https://github.com/HajkD/biomartr/issues">https://github.com/HajkD/biomartr/issues</a></td>
</tr>
<tr>
<td>License:</td>
<td><a href="../../licenses/GPL-3">GPL-3</a></td>
</tr>
<tr>
<td>URL:</td>
<td><a href="https://github.com/HajkD/biomartr">https://github.com/HajkD/biomartr</a></td>
</tr>
<tr>
<td>NeedsCompilation:</td>
<td>yes</td>
</tr>
<tr>
<td>Citation:</td>
<td><a href="citation.html">biomartr citation info</a> </td>
</tr>
<tr>
<td>Materials:</td>
<td><a href="README.html">README</a> <a href="news.html">NEWS</a> </td>
</tr>
<tr>
<td>CRAN&nbsp;checks:</td>
<td><a href="../../checks/check_results_biomartr.html">biomartr results</a></td>
</tr>
</table>
<h4>Downloads:</h4>
<table summary="Package biomartr downloads">
<tr>
<td> Reference&nbsp;manual: </td>
<td> <a href="biomartr.pdf"> biomartr.pdf </a> </td>
</tr>
<tr>
<td>Vignettes:</td>
<td>
<a href="vignettes/BioMart_Examples.html">Ensembl BioMart Examples</a><br/>
<a href="vignettes/Database_Retrieval.html">NCBI Database Retrieval</a><br/>
<a href="vignettes/Functional_Annotation.html">Functional Annotation</a><br/>
<a href="vignettes/Introduction.html">Introduction</a><br/>
<a href="vignettes/MetaGenome_Retrieval.html">Meta-Genome Retrieval</a><br/>
<a href="vignettes/Sequence_Retrieval.html">Sequence Retrieval</a><br/>
</td>
</tr>
<tr>
<td> Package&nbsp;source: </td>
<td> <a href="../../../src/contrib/biomartr_0.4.0.tar.gz"> biomartr_0.4.0.tar.gz </a> </td>
</tr>
<tr>
<td> Windows&nbsp;binaries: </td>
<td> r-devel: <a href="../../../bin/windows/contrib/3.5/biomartr_0.4.0.zip">biomartr_0.4.0.zip</a>, r-release: <a href="../../../bin/windows/contrib/3.4/biomartr_0.4.0.zip">biomartr_0.4.0.zip</a>, r-oldrel: <a href="../../../bin/windows/contrib/3.3/biomartr_0.4.0.zip">biomartr_0.4.0.zip</a> </td>
</tr>
<tr>
<td> OS&nbsp;X&nbsp;El&nbsp;Capitan&nbsp;binaries: </td>
<td> r-release: not available </td>
</tr>
<tr>
<td> OS&nbsp;X&nbsp;Mavericks&nbsp;binaries: </td>
<td> r-oldrel: <a href="../../../bin/macosx/mavericks/contrib/3.3/biomartr_0.4.0.tgz">biomartr_0.4.0.tgz</a> </td>
</tr>
<tr>
<td> Old&nbsp;sources: </td>
<td> <a href="../../../src/contrib/Archive/biomartr"> biomartr archive </a> </td>
</tr>
</table>
<h4>Linking:</h4>
<p>Please use the canonical form
<a href="https://CRAN.R-project.org/package=biomartr"><samp>https://CRAN.R-project.org/package=biomartr</samp></a>
to link to this page.</p>
</body>
</html>
