<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>CRAN - Package indelmiss</title>
<link rel="stylesheet" type="text/css" href="../../CRAN_web.css" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<style type="text/css">
  table td { vertical-align: top; }
</style>
</head>
<body>
<h2>indelmiss: Insertion Deletion Analysis While Accounting for Possible
Missing Data</h2>
<p>Genome-wide gene insertion and deletion rates can be modelled in a maximum 
    likelihood framework with the additional flexibility of modelling potential missing 
    data using the models included within. These models simultaneously estimate insertion 
    and deletion (indel) rates of gene families and proportions of "missing" data for 
    (multiple) taxa of interest. The likelihood framework is utilized for parameter 
    estimation. A phylogenetic tree of the taxa and gene presence/absence patterns 
    (with data ordered by the tips of the tree) are required. For more details, see 
    Utkarsh J. Dang, Alison M. Devault, Tatum D. Mortimer, Caitlin S. Pepperell, 
    Hendrik N. Poinar, G. Brian Golding (2016). Gene insertion deletion analysis 
    while accounting for possible missing data. Genetics (accepted).</p>
<table summary="Package indelmiss summary">
<tr>
<td>Version:</td>
<td>1.0.7</td>
</tr>
<tr>
<td>Depends:</td>
<td>R (&ge; 2.10)</td>
</tr>
<tr>
<td>Imports:</td>
<td><a href="../Rcpp/index.html">Rcpp</a> (&ge; 0.11.2), <a href="../ape/index.html">ape</a> (&ge; 3.2), <a href="../numDeriv/index.html">numDeriv</a> (&ge; 2012.9.1), <a href="../phangorn/index.html">phangorn</a> (&ge; 1.99.13)</td>
</tr>
<tr>
<td>LinkingTo:</td>
<td><a href="../Rcpp/index.html">Rcpp</a></td>
</tr>
<tr>
<td>Suggests:</td>
<td><a href="../testthat/index.html">testthat</a></td>
</tr>
<tr>
<td>Published:</td>
<td>2016-08-22</td>
</tr>
<tr>
<td>Author:</td>
<td>Utkarsh J. Dang and G. Brian Golding</td>
</tr>
<tr>
<td>Maintainer:</td>
<td>Utkarsh J. Dang  &#x3c;&#x75;&#x64;&#x61;&#x6e;&#x67;&#x20;&#x61;&#x74;&#x20;&#x62;&#x69;&#x6e;&#x67;&#x68;&#x61;&#x6d;&#x74;&#x6f;&#x6e;&#x2e;&#x65;&#x64;&#x75;&#x3e;</td>
</tr>
<tr>
<td>License:</td>
<td><a href="../../licenses/GPL-2">GPL-2</a> | <a href="../../licenses/GPL-3">GPL-3</a> [expanded from: GPL (&ge; 2)]</td>
</tr>
<tr>
<td>NeedsCompilation:</td>
<td>yes</td>
</tr>
<tr>
<td>Citation:</td>
<td><a href="citation.html">indelmiss citation info</a> </td>
</tr>
<tr>
<td>Materials:</td>
<td><a href="NEWS">NEWS</a> </td>
</tr>
<tr>
<td>CRAN&nbsp;checks:</td>
<td><a href="../../checks/check_results_indelmiss.html">indelmiss results</a></td>
</tr>
</table>
<h4>Downloads:</h4>
<table summary="Package indelmiss downloads">
<tr>
<td> Reference&nbsp;manual: </td>
<td> <a href="indelmiss.pdf"> indelmiss.pdf </a> </td>
</tr>
<tr>
<td> Package&nbsp;source: </td>
<td> <a href="../../../src/contrib/indelmiss_1.0.7.tar.gz"> indelmiss_1.0.7.tar.gz </a> </td>
</tr>
<tr>
<td> Windows&nbsp;binaries: </td>
<td> r-devel: <a href="../../../bin/windows/contrib/3.5/indelmiss_1.0.7.zip">indelmiss_1.0.7.zip</a>, r-release: <a href="../../../bin/windows/contrib/3.4/indelmiss_1.0.7.zip">indelmiss_1.0.7.zip</a>, r-oldrel: <a href="../../../bin/windows/contrib/3.3/indelmiss_1.0.7.zip">indelmiss_1.0.7.zip</a> </td>
</tr>
<tr>
<td> OS&nbsp;X&nbsp;El&nbsp;Capitan&nbsp;binaries: </td>
<td> r-release: <a href="../../../bin/macosx/el-capitan/contrib/3.4/indelmiss_1.0.7.tgz">indelmiss_1.0.7.tgz</a> </td>
</tr>
<tr>
<td> OS&nbsp;X&nbsp;Mavericks&nbsp;binaries: </td>
<td> r-oldrel: <a href="../../../bin/macosx/mavericks/contrib/3.3/indelmiss_1.0.7.tgz">indelmiss_1.0.7.tgz</a> </td>
</tr>
</table>
<h4>Linking:</h4>
<p>Please use the canonical form
<a href="https://CRAN.R-project.org/package=indelmiss"><samp>https://CRAN.R-project.org/package=indelmiss</samp></a>
to link to this page.</p>
</body>
</html>
