<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>CRAN - Package mixOmics</title>
<link rel="stylesheet" type="text/css" href="../../CRAN_web.css" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<style type="text/css">
  table td { vertical-align: top; }
</style>
</head>
<body>
<h2>mixOmics: Omics Data Integration Project</h2>
<p>Multivariate methods are well suited to large omics data sets where the number of variables (e.g. genes, proteins, metabolites) is much larger than the number of samples (patients, cells, mice). They have the appealing properties of reducing the dimension of the data by using instrumental variables (components), which are defined as combinations of all variables. Those components are then used to produce useful graphical outputs that enable better understanding of the relationships and correlation structures between the different data sets that are integrated. mixOmics offers a wide range of multivariate methods for the exploration and integration of biological datasets with a particular focus on variable selection. The package proposes several sparse multivariate models we have developed to identify the key variables that are highly correlated, and/or explain the biological outcome of interest. The data that can be analysed with mixOmics may come from high throughput sequencing technologies, such as omics data (transcriptomics, metabolomics, proteomics, metagenomics etc) but also beyond the realm of omics (e.g. spectral imaging). The methods implemented in mixOmics can also handle missing values without having to delete entire rows with missing data. A non exhaustive list of methods include generalised Canonical Correlation Analysis, sparse Partial Least Squares and sparse Discriminant Analysis. Recently we implemented integrative methods to combine multiple data sets: horizontal integration with regularised Generalised Canonical Correlation Analysis and vertical integration with multi-group Partial Least Squares.</p>
<table summary="Package mixOmics summary">
<tr>
<td>Version:</td>
<td>6.1.2</td>
</tr>
<tr>
<td>Depends:</td>
<td>R (&ge; 2.10), <a href="../MASS/index.html">MASS</a>, <a href="../lattice/index.html">lattice</a>, <a href="../ggplot2/index.html">ggplot2</a></td>
</tr>
<tr>
<td>Imports:</td>
<td><a href="../igraph/index.html">igraph</a>, <a href="../rgl/index.html">rgl</a>, <a href="../ellipse/index.html">ellipse</a>, <a href="../corpcor/index.html">corpcor</a>, <a href="../RColorBrewer/index.html">RColorBrewer</a>, <a href="../plyr/index.html">plyr</a>, parallel, <a href="../dplyr/index.html">dplyr</a>, <a href="../tidyr/index.html">tidyr</a>, <a href="../reshape2/index.html">reshape2</a>, methods</td>
</tr>
<tr>
<td>Published:</td>
<td>2017-02-27</td>
</tr>
<tr>
<td>Author:</td>
<td>Kim-Anh Le Cao, Florian Rohart, Ignacio Gonzalez, Sebastien Dejean with key contributors Benoit Gautier, Francois Bartolo
        and contributions from Pierre Monget, Jeff Coquery, FangZou Yao, Benoit Liquet. </td>
</tr>
<tr>
<td>Maintainer:</td>
<td>Kim-Anh Le Cao  &#x3c;&#x6b;&#x2e;&#x6c;&#x65;&#x63;&#x61;&#x6f;&#x20;&#x61;&#x74;&#x20;&#x75;&#x71;&#x2e;&#x65;&#x64;&#x75;&#x2e;&#x61;&#x75;&#x3e;</td>
</tr>
<tr>
<td>BugReports:</td>
<td><a href="https://bitbucket.org/klecao/package-mixomics/issues">https://bitbucket.org/klecao/package-mixomics/issues</a></td>
</tr>
<tr>
<td>License:</td>
<td><a href="../../licenses/GPL-2">GPL-2</a> | <a href="../../licenses/GPL-3">GPL-3</a> [expanded from: GPL (&ge; 2)]</td>
</tr>
<tr>
<td>URL:</td>
<td><a href="http://www.mixOmics.org">http://www.mixOmics.org</a></td>
</tr>
<tr>
<td>NeedsCompilation:</td>
<td>no</td>
</tr>
<tr>
<td>Materials:</td>
<td><a href="README">README</a> <a href="NEWS">NEWS</a> </td>
</tr>
<tr>
<td>CRAN&nbsp;checks:</td>
<td><a href="../../checks/check_results_mixOmics.html">mixOmics results</a></td>
</tr>
</table>
<h4>Downloads:</h4>
<table summary="Package mixOmics downloads">
<tr>
<td> Reference&nbsp;manual: </td>
<td> <a href="mixOmics.pdf"> mixOmics.pdf </a> </td>
</tr>
<tr>
<td> Package&nbsp;source: </td>
<td> <a href="../../../src/contrib/mixOmics_6.1.2.tar.gz"> mixOmics_6.1.2.tar.gz </a> </td>
</tr>
<tr>
<td> Windows&nbsp;binaries: </td>
<td> r-devel: <a href="../../../bin/windows/contrib/3.5/mixOmics_6.1.2.zip">mixOmics_6.1.2.zip</a>, r-release: <a href="../../../bin/windows/contrib/3.4/mixOmics_6.1.2.zip">mixOmics_6.1.2.zip</a>, r-oldrel: <a href="../../../bin/windows/contrib/3.3/mixOmics_6.1.2.zip">mixOmics_6.1.2.zip</a> </td>
</tr>
<tr>
<td> OS&nbsp;X&nbsp;El&nbsp;Capitan&nbsp;binaries: </td>
<td> r-release: <a href="../../../bin/macosx/el-capitan/contrib/3.4/mixOmics_6.1.2.tgz">mixOmics_6.1.2.tgz</a> </td>
</tr>
<tr>
<td> OS&nbsp;X&nbsp;Mavericks&nbsp;binaries: </td>
<td> r-oldrel: <a href="../../../bin/macosx/mavericks/contrib/3.3/mixOmics_6.1.2.tgz">mixOmics_6.1.2.tgz</a> </td>
</tr>
<tr>
<td> Old&nbsp;sources: </td>
<td> <a href="../../../src/contrib/Archive/mixOmics"> mixOmics archive </a> </td>
</tr>
</table>
<h4>Reverse dependencies:</h4>
<table summary="Package mixOmics reverse dependencies">
<tr>
<td>Reverse&nbsp;depends:</td>
<td><a href="../bootsPLS/index.html">bootsPLS</a>, <a href="../R2GUESS/index.html">R2GUESS</a>, <a href="../sgPLS/index.html">sgPLS</a>, <a href="../YuGene/index.html">YuGene</a></td>
</tr>
<tr>
<td>Reverse&nbsp;imports:</td>
<td><a href="../plsRcox/index.html">plsRcox</a>, <a href="../RVAideMemoire/index.html">RVAideMemoire</a></td>
</tr>
<tr>
<td>Reverse&nbsp;suggests:</td>
<td><a href="../PLSbiplot1/index.html">PLSbiplot1</a></td>
</tr>
</table>
<h4>Linking:</h4>
<p>Please use the canonical form
<a href="https://CRAN.R-project.org/package=mixOmics"><samp>https://CRAN.R-project.org/package=mixOmics</samp></a>
to link to this page.</p>
</body>
</html>
